docs/docs/dev/documentation/http-api.md

```diff
@@ -63,10 +63,14 @@ By default the API Server is listening on port `5681` (HTTP) and on `5682` (HTTP
 * `/zones/{name}`
 * `/zones+insights`
 * `/zones+insights/{name}`
-* [`/zonesegresses`](#list-zone-egresses)
-* [`/zonesegresses/{name}`](#get-zone-egress)
-* [`/zonesegressoverviews`](#list-zone-egress-overviews)
-* [`/zonesegressoverviews/{name}`](#get-zone-egress-overview)
+* [`/zone-ingresses`](#list-zone-ingresses)
+* [`/zone-ingresses/{name}`](#get-zone-ingress)
+* [`/zoneingresses+insights`](#list-zone-ingress-overviews)
+* [`/zoneingresses+insights/{name}`](#get-zone-ingress-overview)
+* [`/zone-egresses`](#list-zone-egresses)
```

Comment:
I think the endpoint is `/zoneegresses`, no?
```
❯❯❯ curl localhost:5681/zoneegresses
{
 "total": 0,
 "items": [],
 "next": null
}

❯❯❯ curl localhost:5681/zone-egresses
404: Page Not Found⏎
```

Reply:
It was connected to a change in Kuma API. We agreed that we won't change it now so I've reverted to the previous.